.github/workflows/test.yml

```
@@ -19,6 +19,25 @@ jobs:
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
       - name: Build
         run: make build
+  licenses:
+    name: Licenses
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          fetch-depth: 0
+      - name: Get initial commit
+        id: original-commit
```

```suggestion
        id: initial-commit
```

Note: this requires additional changes in this workflow.